diff --git a/Net.py b/Net.py
index 2fb3064bd..3fa0116ed 100644
--- a/Net.py
+++ b/Net.py
@@ -9,9 +9,10 @@ class double(nn.Module):
         super(double, self).__init__()
         self.net1 = Net()
         self.net2 = Net()
-        self.fc1 = nn.Linear(7*7*32, 400)
-        self.fc2 = nn.Linear(400, 100)
-        self.fc3 = nn.Linear(100, 19)
+        self.fc1 = nn.Linear(7*7*32, 800)
+        self.fc2 = nn.Linear(800, 400)
+        self.fc3 = nn.Linear(400, 100)
+        self.fc4 = nn.Linear(100, 19)
 
     def forward(self, x, y):
         x = self.fc1(self.net1(x))
diff --git a/__pycache__/Net.cpython-36.pyc b/__pycache__/Net.cpython-36.pyc
index d4f1d2c40..8eb0e4a71 100644
Binary files a/__pycache__/Net.cpython-36.pyc and b/__pycache__/Net.cpython-36.pyc differ
diff --git a/train.py b/train.py
index b0db5a8ce..332867bbe 100644
--- a/train.py
+++ b/train.py
@@ -24,14 +24,14 @@ if WANDB == True:
 def main():
 
     epochs = 20
-    batch_size =64
+    batch_size =128
     device_id= [0]
 
 
     train_transform = transforms.Compose(
         [    # 360度ランダムで画像を回転する
             # transforms.Resize((32,32)),
-            transforms.ColorJitter(brightness=0.1, contrast=0.1, saturation=0),
+#            transforms.ColorJitter(brightness=0.1, contrast=0.1, saturation=0),
             transforms.ToTensor(),
             transforms.Normalize((0.5, ), (0.5,))])
 
@@ -58,7 +58,7 @@ def main():
     # optimizer = optim.SGD(net.parameters(), lr = 0.001, momentum=0.9)
     #optimizer = optim.SGD(net.parameters(), lr = 0.008, momentum=0.9, weight_decay = 0.008)
 
-    scheduler = optim.lr_scheduler.StepLR(optimizer, step_size=5, gamma=0.5)
+    #scheduler = optim.lr_scheduler.StepLR(optimizer, step_size=5, gamma=0.5)
     #scheduler = optim.lr_scheduler.ExponentialLR(optimizer, gamma=0.9)
 
     train_log = []
@@ -98,7 +98,7 @@ def main():
 
             sys.stdout.write('\r[Epoch %d/%d] [Batch %d/%d] [Train Loss: %.4f]' % (epoch, epochs, i, len(train_loader), total_train_loss[0]/total_train_loss[1]))
 
-        scheduler.step()
+        #scheduler.step()
         t2 = time.time()
         correct_test = 0
         total_test = 0
diff --git a/weight/classificate.pth b/weight/classificate.pth
index 56d2b5f9e..38177c13b 100644
Binary files a/weight/classificate.pth and b/weight/classificate.pth differ
